site/gdocs/utils.tsx

```diff
@@ -71,6 +71,21 @@ export function renderSpan(
                 {renderSpans(span.children)}
             </a>
         ))
+        .with({ spanType: "span-dod" }, (span) => (
+            <span key={key}>
+                <a
+                    key={key}
```

Comment:
nit: don't need the inner key (or, alternatively, can get rid of the `<span>` wrapper)